.github/workflows/scrape.yml

```diff
@@ -285,7 +281,12 @@ jobs:
 
           git add .
 
-          git commit -m "$(date -u)"
+          git commit -m "$(date -u)" || exit 0
+
+          echo {\"last_updated\":\"$(date --iso-8601=seconds -u)\"} > data/meta.json
+          git add meta.json
+          
+          git commit -a --amend --no-edit
```

Comment:
Why not add the meta.json before the initial commit?  That would bypass the need for the commit amendment.

Reply:
The || block won't trigger as by doing that you ensure there always will be something to add (meta.json)